packages/drivers/bun-pg/src/BunPgDriver.ts

```diff
@@ -140,7 +141,7 @@ async function batchUpdate(txn: TransactionSQL, op: UpdateOp): Promise<void> {
         : "";
 
   const sql = `
-    ${cte}
+    ${buildCteSql(cte)}
```

Comment:
The `cte` variable used to be sql, but is now a `ParsedCteClause` so this gets to back over to sql.